static/js/compose_actions.js

```diff
@@ -6,6 +6,7 @@ exports.blur_textarea = function () {
 
 function hide_box() {
     exports.blur_textarea();
+    $("#private_message_recipient").blur();
```

Comment:
Doesn't `blur_textarea` do this already?
```
exports.blur_textarea = function () {                                                                
    $(".message_comp").find("input, textarea, button").blur();                                       
};                                                                                                   
```

`#private_message_recipient` is inside `.message_comp`...

Reply:
No it doesn't because `#private_message_recipient` is a `div`.

Reply:
Hmm, I think the fix should probably still go into `compose_actions.blue_textarea`, by adding it to the `.find()` selectors, right?

Reply:
I didn't because it's not part of the textarea so blurring it in `blur_textarea` doesn't make sense. I am not even sure why we are exporting this function in the first place ... apparently it is only used in the node test where it is set to `noop`, but removing that line doesn't break the test.

Reply:
Sounds likely it was exported for testing with a previous iteration of the tests.  The intent is clearly to blur the compose box inputs; so perhaps we should rename it as well, but we should make it do what it's written to do.  

The bug was introduced in 3a1bf04a561665cd1b4b439c7bd5922a5a58aea9, which changed that element from an `input` to a `div class="input"`.

Reply:
Alright, moved it to `blur_textarea`.